src/core/namespace.c

```diff
@@ -2369,6 +2369,25 @@ int setup_namespace(
                 drop_unused_mounts(root, mounts, &n_mounts);
         }
 
+        /* We make the host's os-release available via a symlink, so that we can copy it atomically
+         * and readers will never get a half-written version. It is unlikely as it's not updated
+         * often, but better safe than sorry. */
```

Comment:
i'd drop the comment about unlikely. This guess might be misleading, because this might happen precisely in the cases where the os-release is reread, so even if updates only happen every week it might be quite likely to be triggered.

But we simply don't know. If I learnt something in my career is that cases we deem unlikely are often much more likely than we think and vice versa. That's why `_likely_()` and `_unlikely()` are so impossibly hard to set correctly.

Reply:
Sure, removed